crates/bevy_math/src/aspect_ratio.rs

```diff
@@ -10,18 +10,74 @@ use bevy_reflect::Reflect;
 #[cfg_attr(feature = "bevy_reflect", derive(Reflect), reflect(Debug, PartialEq))]
 pub struct AspectRatio(f32);
 
+#[derive(Debug, PartialEq)]
+pub enum AspectRatioError {
+    Zero,
+    Infinite,
+    NaN,
+}
+
 impl AspectRatio {
-    /// Create a new `AspectRatio` from a given `width` and `height`.
+    /// Standard 16:9 aspect ratio
+    pub const SIXTEEN_NINE: Self = Self(16.0 / 9.0);
+    /// Standard 4:3 aspect ratio
+    pub const FOUR_THREE: Self = Self(4.0 / 3.0);
+    /// Standard 21:9 ultrawide aspect ratio
+    pub const ULTRAWIDE: Self = Self(21.0 / 9.0);
+
+    /// Create a new [`AspectRatio`] from a given width and height.
```

Comment:
Now that `new` can panic, we should add to the documentation the cases that causes it to panic.
```suggestion
    /// Create a new [`AspectRatio`] from a given width and height.
    ///
    /// # Panics
    /// Panics when the width and height values are invalid, such a zero or infinite widths/heights, or if either
    /// value is `NaN`.
```